### PR TITLE
feat(webpack-config): bump in order to use dart sass

### PR DESCRIPTION
**Pull request by jrood**
### Overview

bump webpack-config package in order to use dart sass

### PR Checklist

- [ ] ~Related to designs:~
- [ ] ~Related to JIRA ticket: [ABC-123]~
- [x] I have run this code to verify it works
- [ ] ~This PR includes unit tests for the code change~
- [ ] ~This PR includes testing instructions tests for the code change~
- [ ] ~The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories~

#### PR Links and Envs
(no gamut kit alpha package generated)

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6425ac297f8f790acc368668--gamut-preview.netlify.app>https://6425ac297f8f790acc368668--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6425ac297f8f790acc368668>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@64.0.6-alpha.3752ab.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://642ae1e4b233892f5d77a3fe--gamut-preview.netlify.app>https://642ae1e4b233892f5d77a3fe--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/642ae1e4b233892f5d77a3fe>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@64.0.6-alpha.612ae2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://642ae4fc6bcf2c31a050824f--gamut-preview.netlify.app>https://642ae4fc6bcf2c31a050824f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/642ae4fc6bcf2c31a050824f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@64.1.2-alpha.cf7a2f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://642b51493144004bc912b14e--gamut-preview.netlify.app>https://642b51493144004bc912b14e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/642b51493144004bc912b14e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@64.1.3-alpha.ddfa95.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://642dd474011b042139bf527c--gamut-preview.netlify.app>https://642dd474011b042139bf527c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/642dd474011b042139bf527c>Deploy Logs</a></p>

File: yarn.lock [hunk 1]
```diff
@@ -17162,7 +17166,7 @@ node-releases@^2.0.6:
   resolved "https://registry.npmjs.org/node-releases/-/node-releases-2.0.6.tgz#8a7088c63a55e493845683ebf3c828d8c51c5503"
   integrity sha512-PiVXnNuFm5+iYkLBNeq5211hvO38y63T0i2KKh2KnUs3RpzJ+JtODFjkD8yjLwnDkTYF1eKXheUwdssR+NRZdg==
 
-node-sass@^6.0.1, node-sass@^8.0.0:
+node-sass@^6.0.1:
```
Comment (jakemhiller) [inline yarn.lock:17169]: i think if you remove `node-sass` from the `resolutions` field in `package.json` it will be removed completely

**Review by jakemhiller — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (jrood) [reply yarn.lock]: ah, good catch! I just removed it from resolutions, re-ran yarn, and I see that it's gone from the yarn lock also.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@64.1.3-alpha.17c42b.0<br/></pre>

**Pull request merged**